src/Mod/Fem/femcommands/commands.py

```diff
@@ -464,7 +464,7 @@ def __init__(self):
                 "Opens the FreeCAD material editor"
             )
         }
-        self.is_active = "allways"
+        self.is_active = "always"
```

Comment:
This is a variable value. It must be used somewhere else, which should be fixed too

Reply:
@yorikvanhavre there is no other instance of this variable value in the rest of the source AFAICT

Reply:
CC @berndhahnebach

Reply:
Is already fixed with 6ba1bc3